services/src/main/resources/locale/portlet/news/News_en.properties

```
@@ -29,6 +29,9 @@ news.composer.postLater=Post later
 news.composer.choosePostDate=Choose when you want your article to be posted:
 news.composer.schedule=Schedule
 news.composer.scheduleSuccess=Post scheduled is displayed
+news.composer.btn.scheduleArticle=Scheduled article
+news.activity.scheduled=Scheduled
```

put this bundle below with the block news.details
```suggestion
news.details.scheduled=Scheduled
```